WordPress/src/main/res/layout/plugin_browser_activity.xml

```diff
@@ -47,11 +47,15 @@
 
                     <android.support.v7.widget.RecyclerView
                         android:id="@+id/installed_plugins_recycler"
-                        android:layout_width="match_parent"
-                        android:layout_height="wrap_content"
+                        android:clipToPadding="false"
                         android:layout_below="@+id/text_manage"
-                        android:layout_marginBottom="@dimen/margin_small"
-                        android:layout_marginTop="@dimen/margin_small"
+                        android:layout_height="wrap_content"
```

Comment:
Should we introduce a new style for these recycler views since they share most, if not all, of the properties?

Reply:
I added a style in 3edc4a9.